### PR TITLE
Home

### PR DESCRIPTION
**Pull request by santiago-88**
@bmentasti dejo las screenshots de el home.
home1
home2
home3
home4
hom5
Espero tus comentarios!

### CODE REVIEW

File: public/stylesheets/footer.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      margin-left: 150px;
+    }
+  }
+  
+  @media (min-width: 500px) {
+    .showw {
+      margin-right: 30px;
+    }
+  }
+
```
Comment (bmentasti) [inline public/stylesheets/footer.css]: No new line at end of file

File: public/stylesheets/registro.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+.logo2{
+    text-align: left;
+    display: block;
+    margin-left:20px ;
+}
+@media (max-width: 600px) {
+    .logo2 {
+      display: none;
+    }
+  }
```
Comment (bmentasti) [inline public/stylesheets/registro.css]: No newline at end of file

File: views/partials/footer.ejs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+        <div class="copyright">
+          <p>Copyright © 2020-2020 S.L. Todos los derechos reservados.</p>
+        </div>
+      </div>
+    </div>
+    </div>
+
+  </footer>
+  </html>
```
Comment (bmentasti) [inline views/partials/footer.ejs]: No new line at end of file

File: views/partials/header.ejs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+          </form>    
+
+        </div>
+      </div>
+
+      <!--buscador-->
+    </div>
+
+  </header>
```
Comment (bmentasti) [inline views/partials/header.ejs]: No new line at end of file

File: views/registro.ejs [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        <img class="logo2" src="../images/logo222.png" alt="">
+
+        <li><a href="">| Inicio</a></li>
+        <li><a href="">| Volver</a></li>
+        <li><a href="">| Ayuda</a></li>
+        <li><a href="">| Sobre nosotros</a></li>
+    </ul>
+</div>
+
+</body>
```
Comment (bmentasti) [inline views/registro.ejs]: No new line at end of file

**Review by bmentasti — CHANGES_REQUESTED** [review]

**Comment by santiago-88** [comment]
homec1
homec2
homec3
homec4
homec6

**Review by bmentasti — APPROVED** [review]

**Review by bmentasti — APPROVED** [review]
ok

**Pull request merged**